test/performance/kshelper.py

```diff
@@ -51,48 +50,54 @@ def load_kubeconfig(self, kubeconfig, context):
         
         api_client = kubernetes.client.ApiClient()
         core_v1_api = kubernetes.client.CoreV1Api(api_client)
+        app_v1_api  = kubernetes.client.AppsV1Api(api_client)
         custom_objects_api = kubernetes.client.CustomObjectsApi(api_client)
 
         os.remove(tmp_config)
-        return core_v1_api, custom_objects_api
+        return core_v1_api, app_v1_api, custom_objects_api
 
-    def get_crd_time(self, grp, v, pl, name=None, check=False):
+    def get_custom_objects_time(self, fname, grp, v, pl, name=None, ns=None, bindingPolicy=None, check=False, single=True):
         try:
-            if name != None and check == False:
-                api_response = self.custom_objects_api.list_cluster_custom_object(group=grp, version=v, plural=pl, field_selector='metadata.name=' + name)
+            if name == None and ns != None and bindingPolicy != None:
+               api_response = self.custom_objects_api.list_namespaced_custom_object(group=grp, version=v, plural=pl, namespace=ns, label_selector='transport.kubestellar.io/originOwnerReferenceBindingKey=' + bindingPolicy)
+            elif  name != None and ns != None and bindingPolicy != None:
+               api_response = self.custom_objects_api.list_namespaced_custom_object(group=grp, version=v, plural=pl, namespace=ns, label_selector='transport.kubestellar.io/originOwnerReferenceBindingKey=' + bindingPolicy)
+            elif name != None and ns != None and bindingPolicy == None:
+               api_response = self.custom_objects_api.get_namespaced_custom_object(group=grp, version=v, namespace=ns, plural=pl, name=name)
             else:
-                api_response = self.custom_objects_api.list_cluster_custom_object(group=grp, version=v, plural=pl)
+               api_response = self.custom_objects_api.list_cluster_custom_object(group=grp, version=v, plural=pl)
+
+            if name == None or single == False:
+               # Create the output file
+               fout = open(fname, 'a')
 
             object_list = api_response["items"]
```

Comment:
Is this right in the cases where `get` was used instead of `list`?

Reply:
Yes, that's still correct. We still want to store the results in the output file.

Reply:
My question is whether `object_list = api_response["items"]` makes sense when `get` was invoked rather than `list`.

Reply:
ok, I understood it now. I updated the script to better handle this scenario. Thanks!